server/middleware/searchHandler.js

```diff
@@ -27,7 +27,8 @@ async function handleSearch(req, res, next) {
                 NI: formData['niNumber'],
                 PrevHocsRef: formData['PrevHocsRef'],
                 RefType: formData['RefType'],
-                HomeSecInterest: formData['HomeSecInterest'] === 'true' ? true : undefined
+                HomeSecInterest: formData['HomeSecInterest'] === 'true' ? true : undefined,
+                CampaignType: formData['CampaignType']
```

Comment:
should there be unit tests associated with this?

Reply:
just added, thanks 👍